mypy/fastparse.py

```diff
@@ -403,13 +442,12 @@ def translate_module_id(self, id: str) -> str:
         return id
 
     def visit_Module(self, mod: ast3.Module) -> MypyFile:
-        body = self.fix_function_overloads(self.translate_stmt_list(mod.body))
-        ignores = [ti.lineno for ti in mod.type_ignores]
-        ignores.extend(self.extra_type_ignores)
+        self.type_ignores = {ti.lineno for ti in mod.type_ignores}
+        body = self.fix_function_overloads(self.translate_stmt_list(mod.body, module=True))
         return MypyFile(body,
                         self.imports,
                         False,
-                        set(ignores),
+                        set(self.type_ignores),
```

Comment:
I'd add a comment explaining why it's copied here. (If there is a good reason!)

Reply:
Looks like this isn't needed. Removed.